.github/workflows/release-announcements.yml

```
@@ -9,6 +9,11 @@ jobs:
   toot:
     runs-on: ubuntu-latest
     steps:
+      - name: get topics and convert to hashtags
+        id: get_topics
+        run: |
+          curl -s https://nf-co.re/pipelines.json | jq -r '.remote_workflows[] | select(.full_name == "${{ github.repository }}") | .topics[]' | awk '{print "#"$0}' | tr '\n' ' ' >> $GITHUB_OUTPUT
```

```suggestion
          echo "topics=$(curl -s https://nf-co.re/pipelines.json | jq -r '.remote_workflows[] | select(.full_name == "${{ github.repository }}") | .topics[]' | awk '{print "#"$0}' | tr '\n' ' ')"  >> $GITHUB_OUTPUT
```
a bug we already fixed in the dev branch, so it will be in the next template update.